jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -144,13 +144,13 @@ private void invalidParamsCheck(IType type, String methodAnnotTarget, Set<String
                         if (!isSpecialType) {
                             // check that if parameter is not a specialType, it has a @PathParam annotation
                             IAnnotation[] param_annotations = param.getAnnotations();
-                            boolean hasPathParamAnnot = Arrays.asList(param_annotations).stream().anyMatch(
+                             boolean hasPathParamAnnot = Arrays.asList(param_annotations).stream().anyMatch(
```

Comment:
kinda nitpicking but there seems to be an extra space that is making this line improperly intended

Reply:
fixed!